bophono/data/exceptions-kvp.csv

```diff
@@ -10,15 +10,28 @@
 ངག་དབང,ngaw-ang
 སྤྱན་རས་གཟིགས,chenrez-ig
 སྣར་ཐང,narth-ang
+བུམ་ཐང་,bumth-ang
 ཞེ་ཆེན,shech-en
-ཞྭ་ལུ/Cu,shal-u
+ཞྭ་ལུ/Cu,shal-
+ཞྭ་དམར་,sham-ar
+ཞྭ་དམར་པ་,shamarp-a
+འཕྲིན་ལས་ ནོར་བུ་,Thin-ley Norbu
 གཏོང་ལེན,tongl-en
 རིན་པོ་ཆེ/Ce,rinpoch-
 འཕོ་བ/Ca,pow-
 ཐང་ཀ/Ca,tangkh-
-བར་ཆད,barch-é
+བར་དུ,bard-u
+བར་ཆད,barch-e
 བར་དོ/Co,bard-
 གཅོད,ch-öd
+འཆི་མེད་,chim-é
+མདོ་མེད་,dom-é
+རིས་མེད་,rim-é
+ཨ་མེས་,am-é
+ཅོ་ནེ་,chon-é
+བལ་ལྡན་,pald-en
```

Comment:
this should fall under the regular rules, no need for an exception

Reply:
The PDF says:
> The é accent should only be applied to the final e when there is a clear risk of mispronunciation (primarily for English words, such as chimé or domé rather than “chime” or “dome”).

Should I ignore it?

Reply:
no the é thing is good, the comment is specifically about palden :)

Reply:
In that case the regular rules should yield "balden", hence the exception.
Or did I misinterpret the rule for བ ?

Reply:
hmmm ok now I'm lost... why would བལ་ལྡན be phoneticized as palden? Is it in one of the KVP documents? (I couldn't find it)

Reply:
Well, two things:
- I think Palden and Namgyal are not exceptions as they seem to follow the rule they give (and thus have no reason to be in the exception file)
- if my memory is right, Palden is usually for "dpal ldan", not "bal dan"

Reply:
Yep you're right I took the first spelling that kind of matched without thinking.
I removed the exception and tested against དཔལ་ལྡན་ that indeed yields palden.
See latest commit for just that change.

Reply:
thanks! I kind of guessed you used your phonetic match in the dictionary actually, I guess I can see some potential flaws in phonetic search now, which is interesting as I'm working on that these days too :)

Just to digress, it's close to a common problem with Tibetan lexicography that leads to mistranslations (or so I've heard): Tibetan to English dictionaries sometimes contain two meanings for a word, one highly obscure and rare, the other extremely common, but don't indicate that too much, and so translators will just sometimes take the obscure meaning for their translations instead of the obvious one. One idea of modern lexicography is to base that on corpus statistics. For Tibetan we're pretty far from having that, but for phonetics it could be relatively straightforward, you could analyze a big corpus like https://zenodo.org/records/3951503 and return the phonetic matches in their order of frequency in the corpus. I'm pretty sure the first result would have been dpal ldan